sdk/compiler/damlc/lib/DA/Cli/Damlc/Command/MultiIde/Util.hs

```diff
@@ -199,6 +219,12 @@ registerFileWatchersMessage multiPackageHome =
         , LSP.FileSystemWatcher (T.pack $ multiPackageHome </> "**/*.dar") Nothing
         , LSP.FileSystemWatcher (T.pack $ multiPackageHome </> "**/*.daml") Nothing
         ]
+      , LSP.SomeRegistration $ LSP.Registration "MultiIdeOpenFiles" LSP.STextDocumentDidOpen $ LSP.TextDocumentRegistrationOptions $ Just $ LSP.List
```

Comment:
This was missing from the last PR which made the multi-IDE a little smarter about when to spin up IDEs.